### DISCUSSION TITLE
F-Droid update to 1.2.6

### DISCUSSION THREAD

**Discussion by ghost** [rustdesk/rustdesk#8544] [Q&A] [answered]
Hi. When will the latest release 1.2.6 be available on F-Droid?

**Comment by rustdesk** [comment]
Hope some contributor can do this, our member maintaining this has left.

**Reply by Selaron** [reply]
Can you reach them to provide some documentation on how this was done in the past?

**Comment by rustdesk** [comment]
@basilgello Thanks for your effort for this, https://gitlab.com/fdroid/fdroiddata/-/blob/master/metadata/com.carriez.flutter_hbb.yml 

Need to add some changes to make new code get built. 
https://github.com/rustdesk/rustdesk/blob/285e974d1a52c891d5fcc28e963d724e085558bc/.github/workflows/flutter-build.yml#L862
https://github.com/rustdesk/rustdesk/blob/285e974d1a52c891d5fcc28e963d724e085558bc/.github/workflows/flutter-build.yml#L864
Flutter version
https://github.com/rustdesk/rustdesk/blob/285e974d1a52c891d5fcc28e963d724e085558bc/.github/workflows/flutter-build.yml#L19

**Reply by rustdesk** [reply]
I totally forgot about this https://gitlab.com/basilgello/rustdesk-fdroid-versioner, and removed the fdroid.yml by mistake, I will bring it back.

**Comment by basilgello** [comment]
I have not left but I had some issues with aom/vpx for RD. Gonna try solving them this week and update fdroid as well. @rustdesk thanks for summary of code changes!

**Reply by rustdesk** [reply]
Thank you very much.

**Comment by basilgello** [comment, marked as answer]
https://gitlab.com/fdroid/fdroiddata/-/merge_requests/15343 once it is accepted, F-Droid machinery will publish the current release :)

**Reply by rustdesk** [reply]
Thanks

**Comment by rustdesk** [comment]
Note: the F-Droid release does not support native hardware encoder for the time being, please use our github release if you need this feature.

**Comment by Selaron** [comment]
Updated 1.2.6 installed via F-Droid. Thank you!
